src/core/markovProcessGraph.js

```diff
@@ -133,37 +136,69 @@ const SEED_ADDRESS = NodeAddress.append(CORE_NODE_PREFIX, "SEED");
 const SEED_DESCRIPTION = "\u{1f331}"; // U+1F331 SEEDLING
 
 // Node address prefix for epoch nodes.
-const EPOCH_PREFIX = NodeAddress.append(CORE_NODE_PREFIX, "EPOCH");
+const USER_EPOCH_PREFIX = NodeAddress.append(CORE_NODE_PREFIX, "USER_EPOCH");
 
 export type EpochNodeAddress = {|
-  +type: "EPOCH_NODE",
+  +type: "USER_EPOCH",
   +owner: NodeAddressT,
   +epochStart: TimestampMs,
 |};
 
 export function epochNodeAddressToRaw(addr: EpochNodeAddress): NodeAddressT {
```

Comment:
`userEpochNodeAddressToRaw` for consistency?

Reply:
Sure: done.